app/models/power_relay.rb

```diff
@@ -4,6 +4,7 @@ module PowerRelay
   def self.included(base)
     ## validations
     base.validates_presence_of :ip, :port, :username, :password
+    base.validates :auto_logout_minutes, :presence => { :if => :auto_logout }
```

Comment:
Use 1.9 hash syntax. I know there's still some residual of the old syntax, but we should change what we can.

We should probably also do a numericality and greater-than-zero validation.

It might also be nice to convert this to `ActiveSupport::Concern` to clean up the syntax a little.